.fixtures.yml

```diff
@@ -1,4 +1,8 @@
 fixtures:
+  forge_modules:
+    stdlib:
+      repo: "puppetlabs/stdlib"
+      ref: "4.6.0"  
```

Comment:
With the requirement of a minimum of 3.0.0, I think we should test that version in tests.